src/serac/physics/materials/green_saint_venant_thermoelastic.hpp

```diff
@@ -74,7 +74,7 @@ struct GreenSaintVenantThermoelasticMaterial {
 
     state.strain_trace = get_value(trEg);
 
-    return serac::tuple{sigma, C_p * density, s0, q0};
+    return serac::tuple{sigma, C_v * density, s0, q0};
```

Comment:
This material got changed from when I wrote it. The `C_v` is meant to be the volumetric heat capacity (at constant volume), so it shouldn't be multiplied by the density.

The pattern of `rho*c` applies when the `c` is a heat capacity per unit mass (that is, specific heat). Multiplying by density turns it into volumetric heat capacity.

TL;DR: remove the `density` factor on the second output.

Reply:
Thanks! I was wondering about that and about to crack open my materials book to check.